pylot/prediction/flags.py

```diff
@@ -6,6 +6,12 @@
 flags.DEFINE_integer(
     'prediction_num_future_steps', None,
     'Number of future steps outputted by the prediction module.')
+flags.DEFINE_integer(
+    'prediction_radius', 50,
+    'Make predictions for all vehicles within this radius of the ego-vehicle.')
+flags.DEFINE_boolean(
```

Comment:
Can we please have this False by default? I think the planners break if we send predictions for ego. We can adjust them in another PR.